### PR TITLE
Fix/move ribbon plots to r

### PR DESCRIPTION
**Pull request by seb231**

### CODE REVIEW

File: src/R/send-charts.R [hunk 1]
```diff
@@ -362,6 +362,33 @@ for (s2 in settings) {
...
+ribbon <- function(data, title) {
+  ggplot(data, aes(year, upper0)) +
+    geom_ribbon(aes(ymax = upper0, ymin = lower0, fill = "#1b9e77"), alpha = 0.2) +
+    geom_ribbon(aes(ymax = upper1, ymin = lower1, fill = "#d95f02"), alpha = 0.2) +
+    geom_ribbon(aes(ymax = upper2, ymin = lower2, fill = "#7570b3"), alpha = 0.2) +
+    geom_ribbon(aes(ymax = upper3, ymin = lower3, fill = "#e7298a"), alpha = 0.2) +
+    ggtitle(paste(title, "probability by academic year")) +
+    xlab("NCY") +
+    ylab("95% probability interval") +
+    scale_fill_manual(name = "Years", labels = c("2013", "2014", "2015", "2016"), values = c("#1b9e77", "#d95f02", "#7570b3", "#e7298a")) +
```
Comment (mikeyford) [inline src/R/send-charts.R]: I think we probably need to infer this automatically since e.g. the Camden data only has three years of transitions.

**Comment by mikeyford** [comment]
Hey, the *_probability charts don't seem to work for Camden data. Only the joiner_probability is produced but it's blank. I think maybe the number of `geom_ribobn`, colours etc. assume 4 years of transitions and need to be changed aswell maybe?

**Comment by seb231** [comment]
thanks @mikeyford will look into this now

File: src/clj/witan/send/charts.clj [hunk 2]
```diff
@@ -1,159 +0,0 @@
-(ns witan.send.charts
```
Comment (mikeyford) [inline src/clj/witan/send/charts.clj]: oh man...feels good to get rid of this!

**Comment by mikeyford** [comment]
Looks good and tests good locally

**Pull request merged**